example/playstore_scrapper_translator_example.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import sys, json
+from datetime import datetime, timedelta
+
+import pytz
+
+from obsei.analyzer.base_analyzer import AnalyzerRequest
+from obsei.analyzer.classification_analyzer import ClassificationAnalyzerConfig, ZeroShotClassificationAnalyzer
+from obsei.analyzer.translation_analyzer import TranslationAnalyzer, TranslationAnalyzerConfig
+from obsei.misc.utils import DATETIME_STRING_PATTERN
+from obsei.source.playstore_scrapper import PlayStoreScrapperConfig, PlayStoreScrapperSource
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+source = PlayStoreScrapperSource()
+
+
+def source_fetch():
+    since_time = datetime.utcnow().astimezone(pytz.utc) + timedelta(days=-1)
+    source_config = PlayStoreScrapperConfig(
+        countries=["us"],
+        package_name="com.color.apps.hindikeyboard.hindi.language",
+        lookup_period=since_time.strftime(DATETIME_STRING_PATTERN),  # todo should be optional
+        max_count=5
+    )
+    return source.lookup(source_config)
+
+
+def translate_text(text_list):
+    translate_analyzer = TranslationAnalyzer(
+        model_name_or_path="Helsinki-NLP/opus-mt-hi-en"
+    )
+    source_responses = [AnalyzerRequest(processed_text=text.processed_text, source_name="sample") for text in text_list]
```

Comment:
What do you think of having single model `AnalyzerData` which only will be carried forward across states?
Mainly I was thinking it will help in DAG based workflow.
https://github.com/lalitpagaria/obsei/issues/8
Like this https://haystack.deepset.ai/docs/latest/pipelinesmd